base/parallel/remotecall.jl

```diff
@@ -280,12 +280,12 @@ function serialize(s::AbstractSerializer, rr::AbstractRemoteRef, addclient)
     invoke(serialize, Tuple{AbstractSerializer, Any}, s, rr)
 end
 
-function deserialize{T<:Future}(s::AbstractSerializer, t::Type{T})
+function deserialize(s::AbstractSerializer, t::Type{<:Future})
     f = deserialize_rr(s,t)
     Future(f.where, RRID(f.whence, f.id), f.v) # ctor adds to client_refs table
 end
 
-function deserialize{T<:RemoteChannel}(s::AbstractSerializer, t::Type{T})
+function deserialize(s::AbstractSerializer, t::Type{<:RemoteChannel})
```

Comment:
you had also converted `function deserialize(s::ClusterSerializer, t::Type{T}) where T <: CapturedException` in an earlier version, which is now in clusterserialize.jl